.github/workflows/daily-generate.yml

```diff
@@ -25,13 +38,31 @@ jobs:
       - name: Install Python packages
         run: venv/bin/python -m pip install -c requirements.txt -e .[server]
 
-      - name: Bulk Generate
-        run: |
-          source venv/bin/activate
-          ./tools/bulk_generate_for_discord.sh
+      - name: Bulk Generate (Cave Story)
+        run: venv/bin/python tools/bulk_generate_for_discord.py --game cave_story
```

Comment:
can we run this for both Starter Preset and CSR Classic? both are supported as defaults and should both ideally be tested by this

Reply:
Can we do that later? I'm not even confident I didn't mess up these cron things

Reply:
would it be anything more than just adding a second line to `run` here with `--preset "CSR Classic"`?

Reply:
If you want to run both together yeah. I think CS generates faster so that might be fine? I don't want to keep the agent locked for too long as it can prevent CI for the other repositories

Reply:
yeah, running both together. CS is quick to generate and if it's really a problem I'd rather have 50 of each at the same time than 100 of each at different times